app/src/main/java/com/sildev/thecocktail/base/BaseViewHolder.kt

```diff
@@ -4,8 +4,20 @@ import androidx.recyclerview.widget.RecyclerView
 import androidx.viewbinding.ViewBinding
 
 abstract class BaseViewHolder<T, VB : ViewBinding>(
-    open val binding: VB
+    private val binding: VB, click: (T) -> Unit
```

Comment:
```suggestion
    private val binding: VB, click: (T) -> Unit = { _ -> }
```
như này thì sau này có cái nào không cần click thì mình không cần truyền tham số thứ 2 nữa